ol-fyi-local-infra/compose.yml

```diff
@@ -40,6 +40,17 @@ services:
       - 6379:6379
       # - 8001:8001
 
+  minio:
+    image: minio/minio
+    command: server /data
+    environment:
+      MINIO_ROOT_USER: minioadmin
+      MINIO_ROOT_PASSWORD: minioadmin123
+    volumes:
+      - /mnt/data:/data
```

Comment:
replace with `- ./volumes/minio/data:/data`

Reply:
Acknowledged and pushed